khl/client.py

```diff
@@ -363,3 +369,29 @@ async def fetch_blocked_friends(self) -> List[Friend]:
 
     async def start(self):
         await asyncio.gather(self.handle_pkg(), self.gate.run(self._pkg_queue))
+
+    async def message_view(self, msg_id: str = '') -> Dict:
+        """get message detail by message id"""
+        if len(msg_id) > 0:
+            return await self.gate.exec_req(api.Message.view(msg_id))
+
+    async def list_messages(self,
+                            target_id: str = None,
```

Comment:
1. it is user-friendlier that expose wrapped concepts to user, rather than underlying mechanism

```suggestion
                            channel: Union[Channel, str],
```

user can pass channel obj or raw channel id as wish